modules/installation-user-defined-tags-azure.adoc

```diff
@@ -0,0 +1,109 @@
+// Module included in the following assemblies:
+// * installing/installing_azure/installing-azure-customizations.adoc
+
+:_content-type: CONCEPT
+[id="installing-azure-user-defined-tags_{context}"]
+= User-defined tags for Azure
+
+In {prodcut-ttitle} you can use the Azure user-defined tags for grouping resources and for managing resource access and cost. You can define tags on the Azure resources in the `install-config.yaml` file only during {product-title} cluster creation. You cannot modify the user-defined Azure tags after cluster creation.
```

Comment:
I did share "In {prodcut-ttitle} you can use the Azure user-defined tags for grouping resources and for managing resource access and cost" to to documented, but would it better to provide a link to Azure tags, rather than us mentioning it here?

Reply:
Yes, we can provide a link here - Do you have the link for the azure tags from azure documentation?

Reply:
`https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/tag-resources?tabs=json` can be referred for more information on Azure Tags.

Reply:
Addressed